### ISSUE TITLE
Book Genre Prediction

### ISSUE THREAD

**Issue by pavitraag** [Niketkumardheeryan/ML-CaPsule#964]
# Book Genre Prediction

## Goal
The goal of this project is to make a prediction model which will predict genre of the book. 

## DATASET
The dataset which is used in this project, is CMU Book Summary Dataset. Here is the link of the dataset : http://www.cs.cmu.edu/~dbamman/booksummaries.html. 

## LIBRARIES NEEDED

- Pandas
- Numpy
- Sklearn
- Matplotlib
- nltk

## Name of Models used
-Support Vector Classifier
-Decision Tree Classifier
-Random Forest Classifier
-Gradient Boosting Classifier
-LightGBM Classifier

**Comment by pavitraag** [comment]
@invigorzz313 Can u please assign me this

**Comment by Niketkumardheeryan** [comment]
@pavitraag these of kind model already there
